contracts/ChildChainGaugeInjectoooor.sol

```diff
@@ -387,9 +404,26 @@ contract ChildChainGaugeInjector is ConfirmedOwner, Pausable, KeeperCompatibleIn
         _unpause();
     }
 
+    /**
```

Comment:
L94: The duplicate check in setRecipientList inspects a storage var (s_targets) inside a loop. It is much more efficient to do this by comparing memory var (gaugeAddresses) elements inside a nested loop.

Reply:
Given the fact that we are not validating if addresses are sorted in both arrays, I suggest leaving this as it is